tensorboard/plugins/image/tf_image_dashboard/tf-image-loader.html

```diff
@@ -63,11 +63,13 @@
         selected$="[[_expanded]]"
         icon="fullscreen"
         on-tap="_toggleExpanded"
+        title="Make the image fill the whole width. Note that this size may differ from actual."
```

Comment:
Why would someone want to do this instead of showing the image at actual size?

(Also, if we do keep this feature, the tooltip is too verbose; suggest "Fill width".)